### ISSUE TITLE
OmniSharp popups use wrong encoding for non-English characters

### ISSUE THREAD

**Issue by oleg-glushko** [tareqimbasher/NetPad#129]
As the title says, here's an example:

<img width="873" alt="NetPad" src="https://github.com/tareqimbasher/NetPad/assets/44775781/e9684617-98f9-4f30-839b-6d7b50540892">

**Comment by tareqimbasher** [comment]
Thank you for reporting. I assume your machine locale is non-English?

**Comment by oleg-glushko** [comment]
Yes, it uses a Cyrillic encoding.

**Comment by my20002000** [comment]
same problem in chinese 
<img width="545" alt="image" src="https://github.com/tareqimbasher/NetPad/assets/48279819/02ff2dc8-497b-4dea-9b3e-9c368a3bee40">

**Comment by JarmanQian** [comment]
@my20002000 非英语环境似乎都有问题，目前是把 AppData\Local\NetPad\[你的版本]\zh-Hans 文件夹改个名字，这样至少能看到英文提示 :D
![image](https://github.com/tareqimbasher/NetPad/assets/25147306/0b599396-56e9-4f6a-8cb4-7f63ff6f5352)

![image](https://github.com/tareqimbasher/NetPad/assets/25147306/e730173c-129d-4b50-b063-1673cd87a07c)

**Comment by tareqimbasher** [comment]
Didn't mean to close it. v0.7.1 will include a fix that hopefully resolves this. Will keep it open for feedback after release.

**Comment by tareqimbasher** [comment]
v0.7.1 is released. Can you guys please try it out and let me know if your issue is resolved?

**Comment by oleg-glushko** [comment]
The Result window displays characters correctly. But now I have another issue with 0.7.1 - IntelliSense doesn't work, so it's impossible to tell if encoding was fixed. All I get is a temporary popup saying "Loading..."

**Comment by tareqimbasher** [comment]
@oleg-glushko Does intellisense still not work after closing out NetPad entirely and then restarting it? If so, can you check the Log file (open Settings and click on Open App Data Folder, there should be a Logs folder there)

**Comment by oleg-glushko** [comment]
Ah, I see... There's the statically defined SDK in global.json pointing to 7.0.0. When I changed it to 8.0.0, everything started working, including the encoding in pop-ups.

```
2024-05-23T09:44:38.4614042+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: System.InvalidOperationException: Failed to find all versions of .NET Core MSBuild. Call to hostfxr_resolve_sdk2. There may be more details in stderr.
2024-05-23T09:44:38.4614591+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: https://aka.ms/dotnet/sdk-not-found
2024-05-23T09:44:38.4614968+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Learn about SDK resolution:
2024-05-23T09:44:38.4615319+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Install the [7.0.0] .NET SDK or update [C:\Program Files\NetPad\resources\bin\global.json] to match an installed SDK.
2024-05-23T09:44:38.4615703+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Installed SDKs:
2024-05-23T09:44:38.4616161+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: global.json file: C:\Program Files\NetPad\resources\bin\global.json
2024-05-23T09:44:38.4616513+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Requested SDK version: 7.0.0
2024-05-23T09:44:38.4616898+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: A compatible .NET SDK was not found.
2024-05-23T09:44:38.4617636+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: https://aka.ms/dotnet/sdk-not-found
2024-05-23T09:44:38.4618094+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Learn about SDK resolution:
2024-05-23T09:44:38.4618531+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Install the [7.0.0] .NET SDK or update [C:\Program Files\NetPad\resources\bin\global.json] to match an installed SDK.
2024-05-23T09:44:38.4618902+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Installed SDKs:
2024-05-23T09:44:38.4619900+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: global.json file: C:\Program Files\NetPad\resources\bin\global.json
2024-05-23T09:44:38.4620676+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: Requested SDK version: 7.0.0
2024-05-23T09:44:38.4621663+10:00 [ERR] (PID: 22936|OmniSharp.Stdio.OmniSharpStdioServer) OmniSharpServer Error Output: A compatible .NET SDK was not found.
2024-05-23T09:44:38.4632303+10:00 [INF] (PID: 22936|OmniSharp.Stdio.Host) Starting OmniSharp on Windows 10.0.19045.0 (x64)
```

```shell
>dotnet sdk check
.NET SDKs:
Version      Status
------------------------
6.0.422      Up to date.
8.0.205      Up to date.
```

**Comment by tareqimbasher** [comment]
Thank you for the info! I'll resolve that issue in the next release.

**Comment by tareqimbasher** [comment]
This issue is resolved with the v0.7.2 release.
